src/components/checkbox/checkbox.tsx

```diff
@@ -0,0 +1,39 @@
+import React = require('react');
+
```

Comment:
`type` suits better for Props than `interface`. `interface` should be implemented, but `type` just describing a structure (shape) of an object – that is what Props is.